docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+
+services:
+  db:
+    image: mdillon/postgis:10-alpine
+  ashlar-server:
+    build: ./ashlar-server
+    depends_on:
+      - db
+    volumes:
+      - ./ashlar-server:/opt/ashlar-server
+    entrypoint: python manage.py runserver 0.0.0.0:8000
```

Comment:
I would suggest making the entrypoint simply `python manage.py` and having `runserver 0.0.0.0:8000` be the command. Then your `/scripts/update` will simplify down to `docker-compose run migrate`.

Actually, now that I've written that, it seems like it could appear a bit magical to someone who wasn't familiar with Docker, so perhaps having the entrypoint as just `python` might be better, then you'd have `manage.py runserver 0.0.0.0:8000` and `manage.py migrate`, which seem clearer. Pick whichever one you prefer!

Reply:
I think a `python` entrypoint is a great middle ground! It afford lots of flexibility for scripting, even outside of Django management commands.